index.html

```diff
@@ -382,28 +382,39 @@ <h1 class="distinguish text-center mb-4">Contact</h1>
           Do you have questions?</p>
       </div>
       <!-- add a PHP form with a mentor -->
-      <form action="#" class="form form-row text-white text-uppercase w-100">
+      <form action="https://wp.pl" class="form form-row text-white text-uppercase w-100">
```

Comment:
usun ta akcje dla formularza, jest ona zbedna do testowania skryptu JS,

jezeli chcesz sprawdzic czy formularz nie przesyla danych bez walidacji to sprawdzaj sobie URL (po wyslaniu danych do adresu doda sie znak #) oraz strona ci sie przeladuje :)